intro-to-git/intro-to-git-vocab-and-concepts.md

```diff
@@ -81,40 +81,38 @@ In order to use Git to compare versions, sometimes we will need to tell Git to _
 
 We will build commits as we build software; we will create commits, and over time, these commits will represent different versions of code and build a history.
 
-Commits not only represent a version of code, but also they contain a lot of extra information. Every commit can also contain the following information:
+Commits are able to capture more than just a snapshot of our code. They can also contain lots of extra information, including:
 
 - A summary of lines of code that were added, deleted, or changed, compared to the last commit
 - A way to refer to the last commit, or its "parent" commit
 - A unique ID for the commit, often called the commit hash
-- A commit message, which will be a string that describes the commit's contents
+- A commit message, a string describing the contents of the commit
 - The date and time that the commit was made
 
-Large projects will be made of hundreds or thousands of commits.
+Large projects often have hundreds or thousands of commits.
 
 ## Teams Collaborate by Merging Histories
 
-In a work day, we want any number of programmers to be able to write code, contribute commits, and build software.
+Over the course of a work day, a team of programmers writes code, contributes commits, and builds software. In order to work together effectively, we need to contribute our own changes, as well as review changes made by other team members.
 
-Git is a version control system that allows for team collaboration. How?
+We need to collaborate! But how can we do this while we are each working on our own code?
 
-Collaboration in this sense means that everyone can contribute changes to code, and look at those changes of code.
+We can enable team collaboration by using Git! Let's examine how.
 
 Every codebase that a team will collaborate on will have a **repository,** or a project folder that contains the files, versioned with Git. (This repository will likely be hosted on github.com)
 
-Every developer who wants to contribute code will need access to the code and files, the repo's commit history, and a way to reference the team's repo. In order to get these three things, every developer will **clone that repo** onto their own machine.
+Every developer who wants to contribute code will need access to the code and files, the repo's commit history, and a way to reference the team's repo. In order to get these three things, every developer will **clone** that repo onto their own machine.
 
 When a developer clones the repo onto their machine, it will create a project folder on their computer, with all of the files inside of it. Then, each developer will write code, run tests, build features, **get updates from the team**, and **make commits.**
 
-Then, each dev who has built commits will send the commits to the team's repository.
+Each dev who makes commits eventually sends the commits to the team's repository.
```

Comment:
How about leaving the sends as will send.  The prior steps all use future rather than present tense.
```suggestion
Each dev who makes commits eventually will send the commits to the team's repository.
```